src/ui/mainwindow.h

```diff
@@ -305,6 +304,7 @@ class MainWindow : public QMainWindow, public PlatformInterface {
   DeviceView* device_view_;
   SongInfoView* song_info_view_;
   ArtistInfoView* artist_info_view_;
+  QueueManager* queue_manager_;
```

Comment:
But other tabs does not have such a destructor. There is "delete ui" in ~QueueManager(), I thought it is enough.